### DISCUSSION TITLE
Running oVirt on Debian?

### DISCUSSION THREAD

**Discussion by emilfihlman** [oVirt/ovirt-site#3164] [General]
There seem to be some notes about this, but it seems that this hasn't materialised yet?

Is there an active roadmap forward for this or is this fully in the far backseat without any current path/work to make it happen?

I would love to run oVirt on Debian.

Thanks!

**Comment by michalskrivanek** [comment]
far, far, very far backseat. I'm not aware of anyone thinking about it anymore. Supporting CentOS Stream is hard enough (and any help is welcomed)

**Reply by emilfihlman** [reply]
Understandable. Do you know what CentOS/etc specific is used? Ie what prevents this currently?

**Reply by sandrobonazzola** [reply]
There are a few pages on oVirt porting to debian/ubuntu: just quick search google: <https://www.google.com/search?q=ovirt+debian>
